### PR TITLE
SaveState: Various Improvements

### PR DESCRIPTION
**Pull request by kamfretoz**
### Description of Changes

This PR adds an option that allows you to configure both the compression type and also the level for savestate.

Preview:
image

Also updates the base FSUI translation and the glyph ranges.

### Rationale behind Changes

QoL Improvements and freedom of choices.

### Suggested Testing Steps

Test if the savestate are compressed and can be loaded correctly.

### CODE REVIEW

**Comment by kamfretoz** [comment]
Now ready for review and testing.

**Review by AmyRoxwell — APPROVED** [review]
Works on linux! All the compression algorithms work correctly and the compression levels work too.
image

**Review by JordanTheToaster — APPROVED** [review]

File: pcsx2/ImGui/FullscreenUI.cpp [hunk 1]
```diff
@@ -3938,7 +3938,11 @@ void FullscreenUI::DrawGraphicsSettingsPage(SettingsInterface* bsi, bool show_ad
 			s_tv_shaders, std::size(s_tv_shaders), true);
 	}
 
-	static constexpr const char* s_gsdump_compression[] = {FSUI_NSTR("Uncompressed"), FSUI_NSTR("LZMA (xz)"), FSUI_NSTR("Zstandard (zst)")};
+	static constexpr const char* s_gsdump_compression[] = {
+		FSUI_NSTR("Uncompressed"),
```
Comment (F0bes) [inline pcsx2/ImGui/FullscreenUI.cpp:3942]: You have a space at the end of the line.

File: pcsx2/ImGui/FullscreenUI.cpp [hunk 2]
```diff
@@ -3938,7 +3938,11 @@ void FullscreenUI::DrawGraphicsSettingsPage(SettingsInterface* bsi, bool show_ad
 			s_tv_shaders, std::size(s_tv_shaders), true);
 	}
 
-	static constexpr const char* s_gsdump_compression[] = {FSUI_NSTR("Uncompressed"), FSUI_NSTR("LZMA (xz)"), FSUI_NSTR("Zstandard (zst)")};
+	static constexpr const char* s_gsdump_compression[] = {
+		FSUI_NSTR("Uncompressed"), 
+		FSUI_NSTR("LZMA (xz)"),
```
Comment (F0bes) [inline pcsx2/ImGui/FullscreenUI.cpp:3943]: Same here

**Review by F0bes — CHANGES_REQUESTED** [review]
Just a couple of formatting issues.

File: pcsx2/ImGui/FullscreenUI.cpp (hunk 1 above)
Comment (kamfretoz) [reply pcsx2/ImGui/FullscreenUI.cpp]: Fixed.

**Comment by kamfretoz** [comment]
I've added a configurable hotkey to dismiss the savestate selector, as well as shorten the auto-close timeout from 7.5s to just 5s.

I'm open to discussion on what the default hotkey for it will be, for now i've decided to go with Shift+Esc. If anyone has better option, do let me know!

image

**Review by JordanTheToaster — APPROVED** [review]

**Review by F0bes — APPROVED** [review]

**Pull request merged**